youtube_dl/jsinterp/jsgrammar.py

```diff
@@ -1,12 +1,10 @@
 from __future__ import unicode_literals
 
 import re
-from enum import Enum
-# ALERT enum34 package dependency
-# it's backported
 
 
-class Token(Enum):
+class T(object):
```

Comment:
works fine, but debugging is a nightmare due to having only token values, I've changed it to namedtuple, after realizing it's supported in 2.6